Generator/EntitiesGenerator.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Pim\Bundle\DataGeneratorBundle\Generator;
+
+use Symfony\Component\Console\Helper\ProgressBar;
+
+/**
+ * Fixture generator that will dispatch generation to specialized generator
+ *
+ * @author    Pierre Allard <pierre.allard@akeneo.com>
+ * @copyright 2016 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class EntitiesGenerator
```

Comment:
[imho] the `registry` or maybe `chained` notion could be found in the name to explicitly define a chained generator